main.go

```diff
@@ -90,48 +92,51 @@ type Factory struct {
 }
 
 func (factory *Factory) NewPurchaseAgreementModel(context ctx.Context, agreement *interop.NotePurchaseAgreement) (interfaces.IAgreementModel, error) {
-	agreementModel, err := NotePurchaseAgreementInitializer()
-	
-	if err != nil {
-		return nil, err
-	}
+	// agreementModel, err := NotePurchaseAgreementInitializer()
+	// fmt.Printf("NewPurchaseAgreementModel: %v\n", agreementModel)
+	// if err != nil {
+	// 	return nil, err
+	// }
 
-	agreementModel.BaseModel = common.NewBaseModel[interop.NotePurchaseAgreement](agreement, context)
-	agreementModel.SetContext(context)
-	agreementModel.SetPayload(agreement)
+	// agreementModel.BaseModel = common.NewBaseModel[interop.NotePurchaseAgreement](agreement, context)
+	// agreementModel.SetContext(context)
+	// agreementModel.SetPayload(agreement)
 
-	return agreementModel, err
+	return &agreements.NotePurchaseAgreementModel{}, nil
 }
 
 func NewNotePurchaseAgreementModelFactory() interfaces.INotePurchaseAgreementModelFactory {
 	return &Factory{}
 }
 
-func NotePurchaseAgreementInitializer() (*agreements.NotePurchaseAgreementModel, error) {
-	wire.Build(
-		config.NewConfig,
-		data.NewGormDatabase,
-		UserSet,
-		NotePuchaseAgreementSet)
-	return nil, nil
-}
+// func NotePurchaseAgreementInitializer() (*agreements.NotePurchaseAgreementModel, error) {
```

Comment:
The existing models and model factories are not used anymore.

Reply:
feel free to remove unused code.